inference-gateway/docker/Dockerfile.postgres

```diff
@@ -0,0 +1,12 @@
+FROM quay.io/tembo/tembo-local
+
+RUN trunk install pg_timeseries
```

Comment:
Perhaps version lock all the trunk installs

Reply:
This is local dev tooling only.